l10n_vn_account_invoice_number/models/account_invoice.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2006-2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    invoice_number = fields.Integer()
```

Comment:
As help message we should put the Vietnamese name of those 3 fields (and we should also mention the Vietnamese names in the README of the module).

Maybe we should rename the field invoice_number into `tax_invoice_number`, then `tax_invoice_form` and `tax_invoice_serie`